hayk.poghosyan/repo/Lesson_3/exercise_13/exercise_13.py

```diff
@@ -0,0 +1,9 @@
+# 13
+meal_cost = float(input("Enter the cost of a meal: "))
+local_tax_rate_text = float(input("Enter the local tax rate: "))
+tip = (meal_cost * 18)/100
+local_tax_rate = (meal_cost * local_tax_rate_text)/100
+total_cost = tip + local_tax_rate + meal_cost
+print("Tip is: " + str(tip))
```

Comment:
RE: Format the output so that it includes a dollar sign and always displays exactly two decimal places.